src/network/mcpe/NetworkSession.php

```diff
@@ -949,11 +950,12 @@ public function syncAvailableCommands() : void{
 
 	public function onChatMessage(Translatable|string $message) : void{
 		if($message instanceof Translatable){
-			$language = $this->player->getLanguage();
-			if(!$this->server->isLanguageForced()){
+			$language = Server::getInstance()->getLanguage();
+			$namespace = explode('.', $message->getText())[0] ?? '';
+			if(!in_array($namespace, $language->getNamespaces(), true) || !$this->server->isLanguageForced()){
```

Comment:
This condition doesn't make sense. It will send untranslated pocketmine strings if the language isn't forced.

This should be &&, not ||.